packages/airnode-validator/src/api/api.ts

```diff
@@ -40,7 +40,7 @@ const secretNamePattern = /^[A-Z][A-Z0-9_]*$/;
 export const secretNameSchema = z
   .string()
   .regex(secretNamePattern, `Secret name is not a valid. Secret name must match ${secretNamePattern.toString()}`);
-export const secretsSchema = z.record(secretNameSchema, z.string());
+export const secretsSchema = z.record(secretNameSchema, z.string().min(1, { message: 'Secret cannot be empty' }));
```

Comment:
As mentioned [here](https://github.com/api3dao/airnode/pull/1172#discussion_r892315600) I disliked the verbose error message, but I don't mind keeping this.